### PR TITLE
Add responsive video header and include with docs

### PR DESCRIPTION
**Pull request by justintoo**
Follow-up fixes for #781:

1. Submit pull request against the develop branch and not master. 
2. Fix indenting in (2 space vs. 4)
3. Move inline CSS to Sass partial (_sass/_utlities.scss)
4. Replace apostrophes with double quotes ie: <div class="embed-container"> instead of <div class='embed-container'>

Additional enhancements:

1. Refactored embed video from `_includes/page__hero_video.html` into `_includes/responsive_video`
2. Updated `_includes/page__hero_video.html` to use `{% include responsive_video id="..." provider="..." %}`
3. Updated documentation with 1) blog post for layout with header video, and 2) new "Video" section in the Helpers documentation.

Let me know if there's any further modifications required for merging. Thanks! =)

### CODE REVIEW

**Comment by mmistakes** [comment]
Quick look and this looks great. I'll have a closer look and let you know. Appreciate you documenting and adding examples too 😄

**Comment by jhabdas** [comment]
**[Yoink](https://github.com/jhabdas/habd.as/commit/0ed11366e24abd90c631fc0ec3b948a1b387060c#diff-ded1f8f5da9ce9a5f5c261a95c9b199fR1)!**

File: _includes/responsive_video [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% capture video_id %}{{ include.id }}{% endcapture %}
+{% capture video_provider %}{{ include.provider }}{% endcapture %}
+
+<!-- Courtesy of embedresponsively.com //-->
+<div class="responsive-video-container">
+{% if video_provider == "vimeo" %}
+  <iframe src="http://player.vimeo.com/video/{{ video_id }}" frameborder="0" webkitAllowFullScreen mozallowfullscreen allowFullScreen></iframe>
```
Comment (jhabdas) [inline _includes/responsive_video]: Hey should these be protocol relative? This breaks when served over HTTPS

Comment (mmistakes) [inline _includes/responsive_video]: Yes it should be protocol relative.

Comment (jhabdas) [inline _includes/responsive_video]: Incoming pull

Comment (mmistakes) [inline _includes/responsive_video]: Maybe just do `https://`, I just checked the embeds on Vimeo and that's what they default to. The YouTube one is that way as well.

Comment (jhabdas) [inline _includes/responsive_video]: Done

**Pull request merged**